tests/ft_cs.vader

```diff
@@ -0,0 +1,30 @@
+Include: include/setup.vader
+
+Execute (cs: msbuild: errorformat):
+  Save &errorformat
+  let &errorformat = neomake#makers#ft#cs#msbuild().errorformat
+  file FooBar.cs
+
+  lgetexpr "FooBar.cs(21,63): error CS1002: ; expected [Foo\Foobar.csproj]"
+  AssertEqual getloclist(0), [{
+    \ 'lnum': 21,
+    \ 'bufnr': bufnr('%'),
+    \ 'col': 63,
+    \ 'valid': 1,
+    \ 'vcol': 0,
+    \ 'nr': 1002,
+    \ 'type': 'E',
+    \ 'pattern': '',
+    \ 'text': "; expected"}]
+
+  lgetexpr "Program.cs(25,29): warning CS0168: The variable 'ex' is declared but never used [Foo\Foobar.csproj]"
+  AssertEqual getloclist(0), [{
+    \ 'lnum': 25,
+    \ 'bufnr': bufnr('%'),
```

Comment:
@blueyed this test is failing because of this line as you can see on the output from the Travis, the '''bufnr('%')''' should grab the buff number right? Maybe it was because I wrote only one test case, should I split it?
'''
[EXECUTE] (X) [{
'lnum': 25,
'bufnr': 3,
'col': 29,
'valid': 1,
'vcol': 0,
'nr': 168,
'type': 'W',
'pattern': '',
'text': 'The variable ''ex'' is declared but never used'}] 

should be equal to [{
'lnum': 25,
'bufnr': 41,
'col': 29,
'valid': 1,
'vcol': 0,
'nr': 168,
'type': 'W',
'pattern': '',
'text': 'The variable ''ex'' is declared but never used'}]
'''